src/game/scenes/vs.c

```diff
@@ -129,6 +130,10 @@ void cb_vs_destroy_object(object *parent, int id, void *userdata) {
 
 void vs_free(scene *scene) {
     vs_local *local = scene_get_userdata(scene);
+    size_t cache_size = sizeof(local->text_cache) / sizeof(local->text_cache[0]);
```

Comment:
Shouldn't we be freeing here?

Reply:
It looks like we free(local) so it's definitely necessary to destroy the text objects here. I agree there should be a text_objects_free here.